src/main/java/org/springframework/data/elasticsearch/core/IndexedObjectInformation.java

```diff
@@ -49,4 +54,8 @@ public long getSeqNo() {
 	public long getPrimaryTerm() {
 		return primaryTerm;
 	}
+
+	public long getVersion() {
```

Comment:
we should change this - and the other two getters to `@Nullable public Long` and return objects instead of primitives

Reply:
Done